stdlib/hashtbl.ml

```diff
@@ -72,7 +72,8 @@ let rec power_2_above x n =
 
 let create ?(random = !randomized) initial_size =
   let s = power_2_above 16 initial_size in
-  let seed = if random then Random.State.bits (Lazy.force prng) else 0 in
+  let random_state = Domain.DLS.get prng_key in
+  let seed = if random then Random.State.bits random_state else 0 in
```

Comment:
it would be nicer if `DLS.get` only occurred under the `if random` test, to avoid initializing the PRNG when it is not used.

Reply:
Good catch. Fixed.